views/my_packages.package.dt

```diff
@@ -15,151 +15,136 @@ block body
 		- if (latest.type == Json.Type.Object)
 			|  #[strong.badge= latest["version"].opt!string]
 
-	- if (error.length)
-		- import std.string : splitLines;
-		p.redAlert
-			- foreach (ln; error.splitLines)
-				|= ln
-				br
+	.subtabsHeader
+		a.tab(data-tab="general", href="#general") General
+		a.tab(data-tab="categories", href="#categories") Categories
+		a.tab(data-tab="repository", href="#repository") Repository
+		a.tab(data-tab="danger", href="#danger") Advanced
+	section.pkgconfig#tab-general
+		a#general(name="general")
+		h2 General
+
+		- if (error.length)
+			- import std.string : splitLines;
+			p.redAlert
+				- foreach (ln; error.splitLines)
+					|= ln
+					br
+
+		- if (pack["errors"].length)
+			p.redAlert
+				- foreach_reverse(e; pack["errors"])
+					|= e.get!string
+					br
+
+		.sideways
+			aside
+				.inputForm
+					form(method="POST", enctype="multipart/form-data", action="#{req.rootDir}my_packages/#{packageName}/set_logo")
+						- string logoPath = "/packages/" ~ packageName ~ "/logo";
+						img.packageLogo(src=logoPath)
+						p.logoError.error
+						label
+							| Select new Logo (max. 1 MiB, optimal size 512x512)
+							input(name="logo", type="file", accept="image/png,image/jpeg,image/gif,image/bmp", onchange="updateLogoPreview(this)")
+						button#logo-upload-button(type="submit") Upload
+					form(method="POST", enctype="multipart/form-data", action="#{req.rootDir}my_packages/#{packageName}/delete_logo")
+						button.danger(type="submit") Delete
+
+			section.contents
+				p
+					- auto stats = registry.getPackageStats(packageName).downloads;
+					| #[strong= stats.daily.to!string] downloads today
+						br
+					| #[strong= stats.weekly.to!string] downloads this week
+						br
+					| #[strong= stats.monthly.to!string] downloads this month
+						br
+					| #[strong= stats.total.to!string] downloads total
```

Comment:
I guess this could be a separate tab "statistics"?

Reply:
I quite like having the statistics directly when you open the package instead.

Maybe a more dedicated border and some more stats would make this better.